projects/plugins/social/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.10.0-beta - 2023-04-25
+## 1.10.0-beta - 2023-05-02
```

Comment:
Shouldn't this be removing the -beta as well?

Reply:
Yes, I amended that in the release branch too.